src/utils/body.ts

```diff
@@ -52,7 +52,7 @@ export async function readBody<
 export async function readValidatedBody<
   Event extends H3Event,
   S extends StandardSchemaV1,
->(event: Event, validate: S): Promise<InferOutput<S>>;
+>(event: Event, validate: S, error?: ValidateError): Promise<InferOutput<S>>;
```

Comment:
For future API extensions (if we need to support more options), it would be better if 3rd arg is an Options object with `error` or `formatError` as key.